ipa-core/src/config.rs

```diff
@@ -414,6 +415,40 @@ impl Debug for Http2Configurator {
     }
 }
 
+#[derive(Default)]
+pub struct KeyRegistries(Vec<KeyRegistry<PublicKeyOnly>>);
+
+impl KeyRegistries {
+    /// # Panics
+    /// If network file is improperly formatted
+    pub fn init_from(
+        &mut self,
```

Comment:
you don't mutate the state of self, this seems to be a static initializer instead

Reply:
this is copied almost exactly from [up here](https://github.com/private-attribution/ipa/pull/1216/files#diff-b2bf3717000008c1caa689282081c4c56fea43a9e4da1d68dd01c4825a0b2d98L178-L205). I added the `key_id` (because it was previously returning a hardcoded DEFAULT_KEY_ID), but it can probably go.